github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPROpenEvent/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger when PR is opened or commits (hash) changed in it.
+    Triggers a build of a pull request when the pull request is opened or reopened.
```

Comment:
Yeah, there's always a race condition possibility for cron checks, I'd assume...